packages/flutterfire_ui/lib/src/i10n/lang/tr.dart

```diff
@@ -0,0 +1,264 @@
+  @override
+  final String signInText;
+  @override
+  final String registerText;
+  @override
+  final String registerHintText;
+  @override
+  final String signInHintText;
+  @override
+  final String signOutButtonText;
+  @override
+  final String phoneInputLabel;
+  @override
+  final String phoneNumberIsRequiredErrorText;
+  @override
+  final String phoneNumberInvalidErrorText;
+  @override
+  final String profile;
+  @override
+  final String name;
+  @override
+  final String deleteAccount;
+  @override
+  final String passwordIsRequiredErrorText;
+  @override
+  final String confirmPasswordIsRequiredErrorText;
+  @override
+  final String confirmPasswordDoesNotMatchErrorText;
+  @override
+  final String confirmPasswordInputLabel;
+  @override
+  final String forgotPasswordButtonLabel;
+  @override
+  final String forgotPasswordViewTitle;
+  @override
+  final String resetPasswordButtonLabel;
+  @override
+  final String verifyItsYouText;
+  @override
+  final String differentMethodsSignInTitleText;
+  @override
+  final String findProviderForEmailTitleText;
+  @override
+  final String continueText;
+  @override
+  final String countryCode;
+  @override
+  final String codeRequiredErrorText;
+  @override
+  final String invalidCountryCode;
+  @override
+  final String chooseACountry;
+  @override
+  final String enableMoreSignInMethods;
+  @override
+  final String signInMethods;
+  @override
+  final String provideEmail;
+  @override
+  final String goBackButtonLabel;
+  @override
+  final String passwordResetEmailSentText;
+  @override
+  final String forgotPasswordHintText;
+  @override
+  final String emailLinkSignInButtonLabel;
+  @override
+  final String signInWithEmailLinkViewTitleText;
+  @override
+  final String signInWithEmailLinkSentText;
+  @override
+  final String sendLinkButtonLabel;
+  @override
+  final String arrayLabel;
+  @override
+  final String booleanLabel;
+  @override
+  final String mapLabel;
+  @override
+  final String nullLabel;
+  @override
+  final String numberLabel;
+  @override
+  final String stringLabel;
+  @override
+  final String typeLabel;
+  @override
+  final String valueLabel;
+  @override
+  final String cancelLabel;
+  @override
+  final String updateLabel;
+  @override
+  final String northInitialLabel;
+  @override
+  final String southInitialLabel;
+  @override
+  final String westInitialLabel;
+  @override
+  final String eastInitialLabel;
+  @override
+  final String timestampLabel;
+  @override
+  final String latitudeLabel;
+  @override
+  final String longitudeLabel;
+  @override
+  final String geopointLabel;
+  @override
+  final String referenceLabel;
+
+  const TrLocalizations({
+    this.emailInputLabel = 'Email',
+    this.passwordInputLabel = 'Şifre',
+    this.signInActionText = 'Giriş yap',
+    this.registerActionText = 'Kayıt ol',
+    this.signInButtonText = 'Giriş yap',
+    this.registerButtonText = 'Kayıt ol',
+    this.linkEmailButtonText = 'İleri',
```

Comment:
This says email button but the next shows translation for "Next". Is this expeceted?

Reply:
The English version was:
```
this.linkEmailButtonText = 'Next',
```